### ISSUE TITLE
Mac And Linux Pyinstaller executables?

### ISSUE THREAD

**Issue by lukegarland** [jaquavis/GymBot#15]
Maybe use GitHub Actions to automatically generate Pyinstaller executables for all platforms that the devs don't have access too? (e.g. MacOS)

**Comment by jaquavis** [comment]
good idea, i could only get it to work for compiling on windows lol, honestly not sure how to compile it for mac or linux
